src/gantt.ts

```diff
@@ -589,7 +589,7 @@ module powerbi.extensibility.visual {
                             identity: selectionId
                         };
 
-                        task.end = d3.time.day.offset(task.start, task.duration);
```

Comment:
We can't add it. These changes will break backward compatibility for DataView of users. You can add optional settings of visual for it ("day" by default).